app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -50,5 +50,13 @@ def textual_id
     def textual_full_name
       {:label => "Full Name", :value => @record.full_name}
     end
+
+    def textual_os_distribution
+      {:label => "Operating System Distribution", :value => @record.operating_system.try(:distribution)} if @record.operating_system.try(:distribution)
```

Comment:
why call `@record.operating_system.try(:distribution)` 2 times instead of setting it to a local variable?

```ruby
distribution = @record.operating_system.try(:distribution)
{:label => "Operating System Distribution", :value => distribution} if distribution
```